lib/packages/docutils/highlite.nim

```diff
@@ -76,14 +81,17 @@ type
 
 const
   sourceLanguageToStr*: array[SourceLanguage, string] = ["none",
-    "Nim", "C++", "C#", "C", "Java", "Yaml", "Python", "Cmd"]
+    "Nim", "C++", "C#", "C", "Java", "Yaml", "Python", "Cmd", "Console"]
+  sourceLanguageToAlpha*: array[SourceLanguage, string] = ["none",
```

Comment:
Just curious, is there any particular reason that only a subset of languages are "supported"? Couldn't the code block logic just allow any language name to be passed through?

Reply:
Our implementation of `code/code-block` allows any language, when a language is not in this list then a warning is generated and it displayed without syntax highlighting, just monospace.